check/check.go

```diff
@@ -47,6 +48,8 @@ type Params struct {
 	OS string `json:"-"`
 	// hardware architecture of target platform
 	Arch string `json:"-"`
+	// Semantic version of the OS
+	OSVersion string `json:"-"`
```

Comment:
Oops sorry this should be marshalled to JSON!

Reply:
`OS` and `Arch` are not required because they are assigned to `Tags` afterwards.